core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorSplit.java

```diff
@@ -20,6 +20,13 @@
 
 public interface ConnectorSplit
 {
+    /**
+     * Returns true when this ConnectorSplit can be scheduled on any node.
+     * <p>
+     * When true, the split may be scheduled on any node. The addresses returned by {@link #getAddresses()}
```

Comment:
Not sure but `the split may be scheduled on any node` is redundant right ?

Reply:
Yes, made it a bit more succinct now